database/src/main/resources/migrations/V7__make_stac_links_of_existing_output.sql

```diff
@@ -0,0 +1,37 @@
+ALTER TABLE predictions ADD COLUMN output_location_copy jsonb not null default '[]';
+
+UPDATE predictions SET output_location_copy = CASE
+    WHEN right(output_location, 7) = 'geojson'
```

Comment:
I didn't know about this function - nice

Reply:
me neither -- i dug through the string functions page looking for a `last` or something and got lucky :game_die: